packages/components/src/internal/components/domainproperties/NameExpressionGenIdBanner.spec.tsx

```diff
@@ -20,9 +22,11 @@ describe('NameExpressionGenIdBanner', () => {
                     kindName="DataClass"
                     api={getTestAPIWrapper(jest.fn, {
                         domain: getDomainPropertiesTestAPIWrapper(jest.fn, {
-                            hasExistingDomainData: () => Promise.resolve(true),
                             getGenId: () => Promise.resolve(123),
                         }),
+                        entity: getEntityTestAPIWrapper(jest.fn, {
+                            isDataTypeEmpty: () => Promise.resolve(false),
```

Comment:
nit: Consider using `jest.fn().mockResolvedValue(false)` to provide a mock implementation.